packages/pytest-simcore/src/pytest_simcore/helpers/httpx_calls_capture_parameters.py

```diff
@@ -34,15 +34,15 @@ def preprocess_type_(cls, val):
     @model_validator(mode="after")
     @classmethod
     def check_compatibility(cls, values):
-        type_ = values.get("type_")
-        pattern = values.get("pattern")
-        format_ = values.get("format_")
-        anyOf = values.get("anyOf")
-        allOf = values.get("allOf")
-        oneOf = values.get("oneOf")
+        type_ = values.type_
```

Comment:
this should be a root_validator or the pydantic v2 pendant instead. instead of playing with strings, you would have the self.pattern